CRITICAL - 2020-11-26 03:41:45 --> Call to undefined method App\Models\GroupPermissionModel::insert_batch
#0 D:\xampp\htdocs\artwork\app\Controllers\Home.php(268): CodeIgniter\Model->__call('insert_batch', Array)
#1 D:\xampp\htdocs\artwork\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Home->storePermission()
#2 D:\xampp\htdocs\artwork\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#3 D:\xampp\htdocs\artwork\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 D:\xampp\htdocs\artwork\public\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2020-11-26 03:54:15 --> syntax error, unexpected '->' (T_OBJECT_OPERATOR)
#0 D:\xampp\htdocs\artwork\vendor\codeigniter4\framework\system\Autoloader\Autoloader.php(296): CodeIgniter\Autoloader\Autoloader->requireFile('D:\\xampp\\htdocs...')
#1 D:\xampp\htdocs\artwork\vendor\codeigniter4\framework\system\Autoloader\Autoloader.php(258): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers...')
#2 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers...')
#3 [internal function]: spl_autoload_call('App\\Controllers...')
#4 D:\xampp\htdocs\artwork\vendor\codeigniter4\framework\system\CodeIgniter.php(869): class_exists('\\App\\Controller...', true)
#5 D:\xampp\htdocs\artwork\vendor\codeigniter4\framework\system\CodeIgniter.php(394): CodeIgniter\CodeIgniter->startController()
#6 D:\xampp\htdocs\artwork\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 D:\xampp\htdocs\artwork\public\index.php(45): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2020-11-26 04:09:06 --> Cannot call constructor
#0 D:\xampp\htdocs\artwork\vendor\codeigniter4\framework\system\CodeIgniter.php(890): App\Controllers\Home->__construct()
#1 D:\xampp\htdocs\artwork\vendor\codeigniter4\framework\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->createController()
#2 D:\xampp\htdocs\artwork\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 D:\xampp\htdocs\artwork\public\index.php(45): CodeIgniter\CodeIgniter->run()
#4 {main}
CRITICAL - 2020-11-26 04:14:22 --> Call to undefined function App\Controllers\user_id()
#0 D:\xampp\htdocs\artwork\vendor\codeigniter4\framework\system\CodeIgniter.php(890): App\Controllers\Home->__construct()
#1 D:\xampp\htdocs\artwork\vendor\codeigniter4\framework\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->createController()
#2 D:\xampp\htdocs\artwork\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 D:\xampp\htdocs\artwork\public\index.php(45): CodeIgniter\CodeIgniter->run()
#4 {main}
CRITICAL - 2020-11-26 04:36:28 --> Call to a member function userdata() on null
#0 [internal function]: CodeIgniter\Events\Events::Config\{closure}()
#1 D:\xampp\htdocs\artwork\vendor\codeigniter4\framework\system\Events\Events.php(188): call_user_func(Object(Closure))
#2 D:\xampp\htdocs\artwork\vendor\codeigniter4\framework\system\CodeIgniter.php(294): CodeIgniter\Events\Events::trigger('user_logs')
#3 D:\xampp\htdocs\artwork\public\index.php(45): CodeIgniter\CodeIgniter->run()
#4 {main}
CRITICAL - 2020-11-26 04:36:56 --> Call to undefined function Config\get_instance()
#0 [internal function]: CodeIgniter\Events\Events::Config\{closure}()
#1 D:\xampp\htdocs\artwork\vendor\codeigniter4\framework\system\Events\Events.php(188): call_user_func(Object(Closure))
#2 D:\xampp\htdocs\artwork\vendor\codeigniter4\framework\system\CodeIgniter.php(294): CodeIgniter\Events\Events::trigger('user_logs')
#3 D:\xampp\htdocs\artwork\public\index.php(45): CodeIgniter\CodeIgniter->run()
#4 {main}
CRITICAL - 2020-11-26 04:37:28 --> Call to undefined function Config\client_ip()
#0 [internal function]: CodeIgniter\Events\Events::Config\{closure}()
#1 D:\xampp\htdocs\artwork\vendor\codeigniter4\framework\system\Events\Events.php(188): call_user_func(Object(Closure))
#2 D:\xampp\htdocs\artwork\vendor\codeigniter4\framework\system\CodeIgniter.php(294): CodeIgniter\Events\Events::trigger('user_logs')
#3 D:\xampp\htdocs\artwork\public\index.php(45): CodeIgniter\CodeIgniter->run()
#4 {main}
CRITICAL - 2020-11-26 05:44:03 --> syntax error, unexpected '->' (T_OBJECT_OPERATOR), expecting identifier (T_STRING) or variable (T_VARIABLE) or '{' or '$'
#0 D:\xampp\htdocs\artwork\vendor\codeigniter4\framework\system\Autoloader\Autoloader.php(296): CodeIgniter\Autoloader\Autoloader->requireFile('D:\\xampp\\htdocs...')
#1 D:\xampp\htdocs\artwork\vendor\codeigniter4\framework\system\Autoloader\Autoloader.php(258): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers...')
#2 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers...')
#3 [internal function]: spl_autoload_call('App\\Controllers...')
#4 D:\xampp\htdocs\artwork\vendor\codeigniter4\framework\system\CodeIgniter.php(870): class_exists('\\App\\Controller...', true)
#5 D:\xampp\htdocs\artwork\vendor\codeigniter4\framework\system\CodeIgniter.php(395): CodeIgniter\CodeIgniter->startController()
#6 D:\xampp\htdocs\artwork\vendor\codeigniter4\framework\system\CodeIgniter.php(313): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 D:\xampp\htdocs\artwork\public\index.php(45): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2020-11-26 05:46:28 --> syntax error, unexpected ')', expecting '('
#0 D:\xampp\htdocs\artwork\vendor\codeigniter4\framework\system\Autoloader\Autoloader.php(296): CodeIgniter\Autoloader\Autoloader->requireFile('D:\\xampp\\htdocs...')
#1 D:\xampp\htdocs\artwork\vendor\codeigniter4\framework\system\Autoloader\Autoloader.php(258): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers...')
#2 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers...')
#3 [internal function]: spl_autoload_call('App\\Controllers...')
#4 D:\xampp\htdocs\artwork\vendor\codeigniter4\framework\system\CodeIgniter.php(870): class_exists('\\App\\Controller...', true)
#5 D:\xampp\htdocs\artwork\vendor\codeigniter4\framework\system\CodeIgniter.php(395): CodeIgniter\CodeIgniter->startController()
#6 D:\xampp\htdocs\artwork\vendor\codeigniter4\framework\system\CodeIgniter.php(313): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 D:\xampp\htdocs\artwork\public\index.php(45): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2020-11-26 06:45:37 --> Class 'UserInfo' not found
#0 [internal function]: CodeIgniter\Events\Events::Config\{closure}()
#1 D:\xampp\htdocs\artwork\vendor\codeigniter4\framework\system\Events\Events.php(188): call_user_func(Object(Closure))
#2 D:\xampp\htdocs\artwork\vendor\codeigniter4\framework\system\CodeIgniter.php(294): CodeIgniter\Events\Events::trigger('user_logs')
#3 D:\xampp\htdocs\artwork\public\index.php(45): CodeIgniter\CodeIgniter->run()
#4 {main}
CRITICAL - 2020-11-26 06:46:18 --> Class 'UserInfo' not found
#0 [internal function]: CodeIgniter\Events\Events::Config\{closure}()
#1 D:\xampp\htdocs\artwork\vendor\codeigniter4\framework\system\Events\Events.php(188): call_user_func(Object(Closure))
#2 D:\xampp\htdocs\artwork\vendor\codeigniter4\framework\system\CodeIgniter.php(294): CodeIgniter\Events\Events::trigger('user_logs')
#3 D:\xampp\htdocs\artwork\public\index.php(45): CodeIgniter\CodeIgniter->run()
#4 {main}
CRITICAL - 2020-11-26 06:46:19 --> Class 'UserInfo' not found
#0 [internal function]: CodeIgniter\Events\Events::Config\{closure}()
#1 D:\xampp\htdocs\artwork\vendor\codeigniter4\framework\system\Events\Events.php(188): call_user_func(Object(Closure))
#2 D:\xampp\htdocs\artwork\vendor\codeigniter4\framework\system\CodeIgniter.php(294): CodeIgniter\Events\Events::trigger('user_logs')
#3 D:\xampp\htdocs\artwork\public\index.php(45): CodeIgniter\CodeIgniter->run()
#4 {main}
CRITICAL - 2020-11-26 06:46:20 --> Class 'UserInfo' not found
#0 [internal function]: CodeIgniter\Events\Events::Config\{closure}()
#1 D:\xampp\htdocs\artwork\vendor\codeigniter4\framework\system\Events\Events.php(188): call_user_func(Object(Closure))
#2 D:\xampp\htdocs\artwork\vendor\codeigniter4\framework\system\CodeIgniter.php(294): CodeIgniter\Events\Events::trigger('user_logs')
#3 D:\xampp\htdocs\artwork\public\index.php(45): CodeIgniter\CodeIgniter->run()
#4 {main}
CRITICAL - 2020-11-26 06:46:20 --> Class 'UserInfo' not found
#0 [internal function]: CodeIgniter\Events\Events::Config\{closure}()
#1 D:\xampp\htdocs\artwork\vendor\codeigniter4\framework\system\Events\Events.php(188): call_user_func(Object(Closure))
#2 D:\xampp\htdocs\artwork\vendor\codeigniter4\framework\system\CodeIgniter.php(294): CodeIgniter\Events\Events::trigger('user_logs')
#3 D:\xampp\htdocs\artwork\public\index.php(45): CodeIgniter\CodeIgniter->run()
#4 {main}
CRITICAL - 2020-11-26 06:46:20 --> Class 'UserInfo' not found
#0 [internal function]: CodeIgniter\Events\Events::Config\{closure}()
#1 D:\xampp\htdocs\artwork\vendor\codeigniter4\framework\system\Events\Events.php(188): call_user_func(Object(Closure))
#2 D:\xampp\htdocs\artwork\vendor\codeigniter4\framework\system\CodeIgniter.php(294): CodeIgniter\Events\Events::trigger('user_logs')
#3 D:\xampp\htdocs\artwork\public\index.php(45): CodeIgniter\CodeIgniter->run()
#4 {main}
CRITICAL - 2020-11-26 06:46:21 --> Class 'UserInfo' not found
#0 [internal function]: CodeIgniter\Events\Events::Config\{closure}()
#1 D:\xampp\htdocs\artwork\vendor\codeigniter4\framework\system\Events\Events.php(188): call_user_func(Object(Closure))
#2 D:\xampp\htdocs\artwork\vendor\codeigniter4\framework\system\CodeIgniter.php(294): CodeIgniter\Events\Events::trigger('user_logs')
#3 D:\xampp\htdocs\artwork\public\index.php(45): CodeIgniter\CodeIgniter->run()
#4 {main}
CRITICAL - 2020-11-26 06:46:21 --> Class 'UserInfo' not found
#0 [internal function]: CodeIgniter\Events\Events::Config\{closure}()
#1 D:\xampp\htdocs\artwork\vendor\codeigniter4\framework\system\Events\Events.php(188): call_user_func(Object(Closure))
#2 D:\xampp\htdocs\artwork\vendor\codeigniter4\framework\system\CodeIgniter.php(294): CodeIgniter\Events\Events::trigger('user_logs')
#3 D:\xampp\htdocs\artwork\public\index.php(45): CodeIgniter\CodeIgniter->run()
#4 {main}
CRITICAL - 2020-11-26 06:46:21 --> Class 'UserInfo' not found
#0 [internal function]: CodeIgniter\Events\Events::Config\{closure}()
#1 D:\xampp\htdocs\artwork\vendor\codeigniter4\framework\system\Events\Events.php(188): call_user_func(Object(Closure))
#2 D:\xampp\htdocs\artwork\vendor\codeigniter4\framework\system\CodeIgniter.php(294): CodeIgniter\Events\Events::trigger('user_logs')
#3 D:\xampp\htdocs\artwork\public\index.php(45): CodeIgniter\CodeIgniter->run()
#4 {main}
CRITICAL - 2020-11-26 06:51:12 --> Call to a member function getMethod() on null
#0 D:\xampp\htdocs\artwork\vendor\codeigniter4\framework\system\Config\Services.php(726): CodeIgniter\Router\Router->__construct(Object(CodeIgniter\Router\RouteCollection), NULL)
#1 D:\xampp\htdocs\artwork\vendor\codeigniter4\framework\system\Config\BaseService.php(123): CodeIgniter\Config\Services::router(Object(CodeIgniter\Router\RouteCollection), NULL, false)
#2 D:\xampp\htdocs\artwork\vendor\codeigniter4\framework\system\Config\Services.php(718): CodeIgniter\Config\BaseService::getSharedInstance('router', NULL, NULL)
#3 D:\xampp\htdocs\artwork\vendor\codeigniter4\framework\system\Common.php(957): CodeIgniter\Config\Services::router()
#4 D:\xampp\htdocs\artwork\app\Config\Events.php(29): service('router')
#5 [internal function]: CodeIgniter\Events\Events::Config\{closure}()
#6 D:\xampp\htdocs\artwork\vendor\codeigniter4\framework\system\Events\Events.php(188): call_user_func(Object(Closure))
#7 D:\xampp\htdocs\artwork\vendor\codeigniter4\framework\system\CodeIgniter.php(294): CodeIgniter\Events\Events::trigger('user_logs')
#8 D:\xampp\htdocs\artwork\public\index.php(45): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2020-11-26 06:52:19 --> Call to a member function getMethod() on null
#0 D:\xampp\htdocs\artwork\vendor\codeigniter4\framework\system\Config\Services.php(726): CodeIgniter\Router\Router->__construct(Object(CodeIgniter\Router\RouteCollection), NULL)
#1 D:\xampp\htdocs\artwork\vendor\codeigniter4\framework\system\Config\BaseService.php(123): CodeIgniter\Config\Services::router(Object(CodeIgniter\Router\RouteCollection), NULL, false)
#2 D:\xampp\htdocs\artwork\vendor\codeigniter4\framework\system\Config\Services.php(718): CodeIgniter\Config\BaseService::getSharedInstance('router', NULL, NULL)
#3 D:\xampp\htdocs\artwork\vendor\codeigniter4\framework\system\Common.php(957): CodeIgniter\Config\Services::router()
#4 D:\xampp\htdocs\artwork\app\Config\Events.php(29): service('router')
#5 [internal function]: CodeIgniter\Events\Events::Config\{closure}()
#6 D:\xampp\htdocs\artwork\vendor\codeigniter4\framework\system\Events\Events.php(188): call_user_func(Object(Closure))
#7 D:\xampp\htdocs\artwork\vendor\codeigniter4\framework\system\CodeIgniter.php(294): CodeIgniter\Events\Events::trigger('user_logs')
#8 D:\xampp\htdocs\artwork\public\index.php(45): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2020-11-26 07:12:24 --> Call to undefined method CodeIgniter\HTTP\IncomingRequest::uri()
#0 [internal function]: CodeIgniter\Events\Events::Config\{closure}()
#1 D:\xampp\htdocs\artwork\vendor\codeigniter4\framework\system\Events\Events.php(188): call_user_func(Object(Closure))
#2 D:\xampp\htdocs\artwork\vendor\codeigniter4\framework\system\CodeIgniter.php(294): CodeIgniter\Events\Events::trigger('user_logs')
#3 D:\xampp\htdocs\artwork\public\index.php(45): CodeIgniter\CodeIgniter->run()
#4 {main}
CRITICAL - 2020-11-26 07:13:22 --> Call to a member function getMethod() on null
#0 D:\xampp\htdocs\artwork\vendor\codeigniter4\framework\system\Config\Services.php(726): CodeIgniter\Router\Router->__construct(Object(CodeIgniter\Router\RouteCollection), NULL)
#1 D:\xampp\htdocs\artwork\vendor\codeigniter4\framework\system\Config\BaseService.php(123): CodeIgniter\Config\Services::router(Object(CodeIgniter\Router\RouteCollection), NULL, false)
#2 D:\xampp\htdocs\artwork\vendor\codeigniter4\framework\system\Config\Services.php(718): CodeIgniter\Config\BaseService::getSharedInstance('router', NULL, NULL)
#3 D:\xampp\htdocs\artwork\vendor\codeigniter4\framework\system\Common.php(957): CodeIgniter\Config\Services::router()
#4 D:\xampp\htdocs\artwork\app\Config\Events.php(32): service('router')
#5 [internal function]: CodeIgniter\Events\Events::Config\{closure}()
#6 D:\xampp\htdocs\artwork\vendor\codeigniter4\framework\system\Events\Events.php(188): call_user_func(Object(Closure))
#7 D:\xampp\htdocs\artwork\vendor\codeigniter4\framework\system\CodeIgniter.php(294): CodeIgniter\Events\Events::trigger('user_logs')
#8 D:\xampp\htdocs\artwork\public\index.php(45): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2020-11-26 08:22:39 --> Call to undefined function site()
#0 D:\xampp\htdocs\artwork\vendor\codeigniter4\framework\system\View\View.php(246): include()
#1 D:\xampp\htdocs\artwork\vendor\codeigniter4\framework\system\Common.php(1119): CodeIgniter\View\View->render('register', Array, true)
#2 D:\xampp\htdocs\artwork\app\Controllers\Users.php(61): view('register')
#3 D:\xampp\htdocs\artwork\vendor\codeigniter4\framework\system\CodeIgniter.php(919): App\Controllers\Users->register()
#4 D:\xampp\htdocs\artwork\vendor\codeigniter4\framework\system\CodeIgniter.php(405): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#5 D:\xampp\htdocs\artwork\vendor\codeigniter4\framework\system\CodeIgniter.php(313): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 D:\xampp\htdocs\artwork\public\index.php(45): CodeIgniter\CodeIgniter->run()
#7 {main}
